unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs

```diff
@@ -728,4 +728,12 @@ private void UpdateAvatarModelWhenNeeded()
             isAvatarDirty = false;
         }
     }
+
+    private void OnApplicationFocus(bool hasFocus)
+    {
+        if (!hasFocus)
+        {
+            controller.LostFocus();
+        }
+    }
```

Comment:
I have a proposal for this!

Instead of this way I would create a Service ni the `ServiceLocator` so more systems can subscribe to that event.

Reply:
Sounds good!